openslides/core/static/templates/projector.html

```diff
@@ -7,6 +7,7 @@
 <link rel="stylesheet" href="static/css/projector.css">
 <link rel="icon" href="/static/img/favicon.png">
 <script src="static/js/openslides-libs.js"></script>
+<script src="static/js/openslides-libs.worker.js"></script>
```

Comment:
Please add an issue that we have to find another solution for this. It is not best practice to add extra libs and generate extra requests for this. Maybe we can concat this file together with openslides-libs.js.